bats/tests/k8s/wordpress.bats

```diff
@@ -1,11 +1,8 @@
-# Test case 8 & 13
-
-# shellcheck disable=SC2030,SC2031
-# SC2030 (info): Modification of output is local (to subshell caused by @bats test).
-# SC2031 (info): output was modified in a subshell. That change might be lost.
-
 setup() {
     load '../helpers/load'
+    if [ "$ARCH" = "arm64" ]; then
+        skip "Wordpress is not compatible with arm64 architecture. Skipping..."
```

Comment:
It is the Bitname image not being provided in the required format; wordpress itself is compatible with arm64.

```suggestion
        skip "The bitnami wordpress image is not available for arm64 architecture. Skipping..."
```